apps/wildfires/src/app/pages/about/index.tsx

```diff
@@ -11,40 +6,14 @@ export default function About() {
     <>
       <HorizontalLayout className="bg-brand-dark-blue">
         <div className="relative">
-          <div className="max-w-[62.5rem] mx-auto flex flex-col md:flex-row items-stretch">
-            <div className="bg-brand-dark-blue w-full md:w-[60%] py-40 flex items-center  md:py-0">
-              <h1 className="border-l-[10px] border-brand-yellow font-light font-primary md:font-bold pl-4 md:pl-8 text-5xl md:text-[64px] text-white md:leading-[1.2] text-left">
-                About <span className="whitespace-nowrap">LA Disaster</span>{' '}
-                Relief Navigator
+          <div className="max-w-[62.5rem] mx-auto flex flex-col md:flex-row items-stretch min-h-[60vh] py-14 md:py-28">
```

Comment:
maybe
```suggestion
          <div className="max-w-[62.5rem] mx-auto flex flex-col md:flex-row items-stretch min-h-[10vh] py-14 md:py-28">
```

`60vh` is quite huge 
<img width="1037" alt="image" src="https://github.com/user-attachments/assets/4e334337-77e1-4f8f-8fa5-027eb5a51fc7" />